generate_jst_sh_connectors.py

```diff
@@ -633,6 +635,11 @@ def generate_dev(
 
     dev.add_part(Part(dev_name, Manufacturer("JST")))
```

Comment:
Unfortunately the MPNs now have the "JST " prefix too because `dev_name` is used. Maybe better keep `dev_name` without the prefix and add it only when constructing the `Device` object?

(comment also applies to `part_with_suction_cap` below)

Reply:
Good catch!